ui/src/pages/Login/styles.module.css

```diff
@@ -0,0 +1,56 @@
+.container {
+  height: 100vh;
+  width: 100vw;
+  background-color: #e2e2e2;
+  background-image: url('../../images/background.svg');
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.card {
+  background-color: var(--primary-bg);
+  height: 60vh;
+  width: 22vw;
+  border-radius: 40px;
+  filter: drop-shadow(10px 10px 30px rgba(0, 0, 0, 0.25));
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10vh 0 0 0;
+}
+
+.logo {
+  height: 15vh;
+}
+
+.heading {
+  margin-top: 4vh;
+  color: var(--primary-text);
+  font-size: 2rem;
+  font-weight: 800;
+}
+
+.btn {
+  padding: 1.5vh 3vh;
+  border: none;
+  margin-top: 5vh;
+  background-color: var(--secondary-bg);
+  border-radius: 15px;
+  color: var(--primary-bg);
+  font-size: 1.2rem;
+  font-weight: 700;
+  cursor: pointer;
+}
+.footer {
+  margin-top: 5vh;
+  width: 100%;
+  color: var(--secondary-color);
+  background-color: var(--primary-color);
+  height: 10vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0 0 40px 40px;
+}
```

Comment:
Somehow this style leaves space at the bottom.

![image](https://user-images.githubusercontent.com/15676805/173240072-33e75c20-d2ab-4a72-b02a-9a7dabd07e31.png)